modules/apps/dynamic-data-mapping/dynamic-data-mapping-form-renderer/src/main/java/com/liferay/dynamic/data/mapping/form/renderer/internal/DDMFormFieldTemplateContextFactory.java

```diff
@@ -216,35 +216,23 @@ protected DDMFormFieldRenderingContext createDDDMFormFieldRenderingContext(
 
 		int index = 0;
 
-		if (ddmFormFieldValues == null) {
-			ddmFormFieldTemplateContexts.add(
-				createDDMFormFieldTemplateContext(
-					new DDMFormFieldValue() {
-						{
-							setName(_ddmFormFieldName);
-						}
-					},
-					new HashMap<>(), index++, parentDDMFormFieldParameterName));
-		}
```

Comment:
@rodrigopaulino have you removed this `if` statement because the values ​​will always be created by `creatDefaultDDMFormFieldValue` in the `_populate` method?

Reply:
That's correct, @carolmariaabb